.travis.yml

```diff
@@ -3,6 +3,7 @@ node_js:
   - 6
   - 8
   - 9
```

Comment:
Node 9 is no longer supported by the core team (not covered by LTS). Fine to keep here just to be notified if/when we do lose support, but I wouldn't go out of our way to keep supporting it if something does come up.